inventory

```diff
@@ -4,19 +4,19 @@
 #1.1.1.3 ansible_ssh_user=tailf ansible_ssh_pass=password
 
 [production:vars]
-linux_ver=RH6
+linux_ver=Redhat
```

Comment:
Do you not think that we need to know the version within the major SW type?

Reply:
I think yes, you can keep the version. I removed since we are starting working on NSO and we have only one major release per Linux distribution!